Essentials/src/com/earth2me/essentials/Essentials.java

```diff
@@ -381,6 +383,13 @@ public void onDisable() {
         getUserMap().getUUIDMap().shutdown();
 
         HandlerList.unregisterAll(this);
+        while (getBackup().isActive()) {
+            try {
+                Thread.sleep(10);
```

Comment:
I see what you're trying to do, but as Aikar pointed out this would probably be done much better with a future. If that's too much work here, I'd at least change this to only sleep for one second at a time, to avoid superfluous waiting. The main thread doesn't have anything better to do at this point anyways.

Reply:
I agree this should wait on a future. We should also print a message informing that we're waiting for the backup to finish.

Reply:
Good point on the info message, that'd definitely be good feedback to give